ethcore/src/header.rs

```diff
@@ -219,8 +219,8 @@ impl Header {
 		s.append(&self.timestamp);
 		s.append(&self.extra_data);
 		if let Seal::With = with_seal {
-			for b in &self.seal { 
-				s.append_raw(&b, 1); 
+			for b in &self.seal {
```

Comment:
I see... than we have inconsistency in `HeaderView` struct, cause it returns pre-RLP-encoded data. I will update the pr.